db/migrate/20250923072038_add_vaccination_record_discovered_sent_at_to_consents.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddVaccinationRecordDiscoveredSentAtToConsents < ActiveRecord::Migration[
+  8.0
+]
+  def change
+    add_column :consents,
+               :vaccination_record_discovered_sent_at,
```

Comment:
Just wondering about this naming, because it's not just that a vaccination record was discovered, it's specifically that one was found which means the patient is actually already vaccinated. I'm wondering if a name like `patient_already_vaccinated_notification_sent_at` is clearer?